src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -14,27 +14,46 @@
 			@reset-filters="resetFilters"
 			@handle-overlay="handleQuickFiltersOverlay"
 		/>
+		<div
+			class="tw-flex tw-flex-wrap"
+		>
+			<kiva-classic-basic-loan-card
+				v-for="(loan, index) in loans"
+				:key="`loan-${loan.id}`"
+				:item-index="index"
+				:loan-id="loan.id"
+				:exp-label="expLabel"
+				:show-action-button="true"
+				style="margin-right: 16px; max-width: 300px;"
+			/>
+		</div>
 	</div>
 </template>
 
 <script>
 import QuickFilters from '@/components/LoansByCategory/QuickFilters/QuickFilters';
-import { runFacetsQueries, fetchLoanFacets } from '@/util/loanSearch/dataUtils';
+import { runFacetsQueries, fetchLoanFacets, runLoansQuery } from '@/util/loanSearch/dataUtils';
 import { fetchCategories, FLSS_ORIGIN_CATEGORY } from '@/util/flssUtils';
 import { transformIsoCodes } from '@/util/loanSearch/filters/regions';
+import KivaClassicBasicLoanCard from '@/components/LoanCards/KivaClassicBasicLoanCard';
 
 export default {
 	name: 'QuickFiltersSection',
 	components: {
-		QuickFilters
+		QuickFilters,
+		KivaClassicBasicLoanCard
 	},
 	inject: ['apollo'],
 	data() {
 		return {
-			filtersLoaded: false,
+			totalCount: 0,
 			targetedLoanChannelURL: '',
+			filtersLoaded: false,
 			flssLoanSearch: {},
-			loanSearchState: {},
+			loanSearchState: {
+				pageOffset: 0,
+				pageLimit: 3
```

Comment:
Is there a future ticket for a pager? I see we only show 3 loans currently.

Reply:
yes!